Subject: Linux-RT 2.6.24-rc2-rt1
From: http://www.kernel.org/pub/linux/kernel/projects/rt/
Acked-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
From: Andrew Morton <akpm@linux-foundation.org>

arch/i386/kernel/quirks.c: In function 'ich_force_enable_hpet':
arch/i386/kernel/quirks.c:94: warning: 'rcba' is used uninitialized in this function

Cc: Andi Kleen <ak@suse.de>
Cc: Greg KH <greg@kroah.com>
Cc: Ingo Molnar <mingo@elte.hu>
Cc: Thomas Gleixner <tglx@linutronix.de>
Cc: Venkatesh Pallipadi <venkatesh.pallipadi@intel.com>
Cc: Venki Pallipadi <venkatesh.pallipadi@intel.com>
Cc: john stultz <johnstul@us.ibm.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/i386/kernel/quirks.c |    3 ++-
 asm-i386/hpet.h           |    0 
 arch/i386/kernel/quirks.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: linux-2.6.23-rc4/arch/i386/kernel/quirks.c
===================================================================
--- linux-2.6.23-rc4.orig/arch/i386/kernel/quirks.c	2007-08-31 21:51:23.000000000 +0200
+++ linux-2.6.23-rc4/arch/i386/kernel/quirks.c	2007-08-31 21:51:23.000000000 +0200
@@ -83,7 +83,8 @@ void ich_force_hpet_resume(void)
 
 static void ich_force_enable_hpet(struct pci_dev *dev)
 {
-	u32 val, rcba;
+	u32 val;
+	u32 uninitialized_var(rcba);
 	int err = 0;
 
 	if (hpet_address || force_hpet_address)
